### PR TITLE
Add warning about using public classes

### PR DESCRIPTION
**Pull request by kenlyon**
Added a warning about risks of using undocumented classes/functions.

Other changes:

* Changed prereqs links to C# and .NET Standard.
* Fixed some spelling/grammar issues.

AB#40504

### CODE REVIEW

File: docs/mobile/sdk-components-reference.mdx [hunk 1]
```diff
@@ -95,7 +95,7 @@ namespace App1.Components
 
 #### Component Dependencies
 
-Components are instantiated by [dependency injection](sdk-dependency-injection.mdx) when the layout is rendered. This means that a component can also list dependencies that have been registered with autofac, such as the [`ILayoutModel<MapView>`](sdk-component-service-interactions.mdx) in it's constructor, and have them be passed at runtime. This also means that if a class associated with a component has application dependencies, such as a reference to the `MapOperations` for map interactions, then the component will either need to provide that dependency somehow.
+Components are instantiated by [dependency injection](sdk-dependency-injection.mdx) when the layout is rendered. This means that a component can also list dependencies that have been registered with autofac, such as the [`ILayoutModel<MapView>`](sdk-component-service-interactions.mdx) in its constructor, and have them be passed at runtime. This also means that if a class associated with a component has application dependencies, such as a reference to the `MapOperations` for map interactions, then the component will either need to provide that dependency somehow.
```
Comment (jopayne) [inline docs/mobile/sdk-components-reference.mdx:98]: This also means that if a class associated with a component has application dependencies, such as a reference to the `MapOperations` for map interactions, then the component will **either** need to provide that dependency somehow.

Use of 'either' implies that an 'or' will follow... Maybe you don't need the word 'either'?

File: docs/mobile/sdk-components-reference.mdx [hunk 2]
```diff
@@ -130,7 +130,7 @@ Check out this example of a [component with a view model](https://github.com/geo
 
 ## Configuration Models
 
-Every component is bound to specific [**item type**](configuration-app-config-reference.mdx#app-items), and each item type is bound to a class [registered as an `AppItem` with autofac](sdk-dependency-injection.mdx#appitem). This class instantiates itself with values from the app config in it's constructor, acting as a configuration model. The component class then consumes this configuration model, and then passes the appropriate values to the view and view model.
+Every component is bound to specific [**item type**](configuration-app-config-reference.mdx#app-items), and each item type is bound to a class [registered as an `AppItem` with autofac](sdk-dependency-injection.mdx#appitem). This class instantiates itself with values from the app config in its constructor, acting as a configuration model. The component class then consumes this configuration model, and then passes the appropriate values to the view and view model.
```
Comment (jopayne) [inline docs/mobile/sdk-components-reference.mdx:133]: "The component class **then** consumes this configuration model, and then passes the appropriate values to the view and view model."

I'd remove the first instance of 'then'.

File: docs/workflow/snippets/sdk-net-limitations-warning.mdx [hunk 3]
```diff
@@ -0,0 +1,3 @@
+:::note
+We recommend that you only implement the <code>IActivityHandler</code> interface examples provided in the Developer Center. These examples have been verified and are provided to help you customize your Geocortex product to suit your needs. If you choose to use classes or functions not documented here, you may break functionality and/or experience unexpected results.
```
Comment (rcooney) [inline docs/workflow/snippets/sdk-net-limitations-warning.mdx:2]: "If you choose to use classes or functions not documented here" sounds super broad. What we mean is that you should not reference other classes and functions from the various Geocortex Workflow assemblies .

File: docs/mobile/sdk-components-reference.mdx (hunk 2 above)
Comment (kenlyon) [reply docs/mobile/sdk-components-reference.mdx:133]: I was just fixing the spelling of "its" here. Strictly speaking that was out of scope and I'd rather not get sidetracked fixing more.

File: docs/mobile/sdk-components-reference.mdx (hunk 1 above)
Comment (kenlyon) [reply docs/mobile/sdk-components-reference.mdx:98]: As with the first comment, I was only interested in fixing the spelling here.

File: docs/workflow/snippets/sdk-net-limitations-warning.mdx (hunk 3 above)
Comment (kenlyon) [reply docs/workflow/snippets/sdk-net-limitations-warning.mdx]: Changed

Comment (rcooney) [reply docs/workflow/snippets/sdk-net-limitations-warning.mdx]: Am I crazy or does this now read like "you should only implement these exact examples that we are providing on the DevCenter. If you do anything else you are taking a risk." 

We want to make it clear that IActivityHandler is fully supported, but all the other public classes and functions **from Geocortex Workflow** are not.

Comment (kenlyon) [reply docs/workflow/snippets/sdk-net-limitations-warning.mdx]: That's what I had meant by "from the various Geocortex Workflow assemblies". I'm open to suggestions if there's a better way to word it.

Comment (rcooney) [reply docs/workflow/snippets/sdk-net-limitations-warning.mdx]: Turns out I was crazy, and looking at the outdated code. All good.

**Pull request merged**